scripts/build-nhc-snr.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+set -e
+
+export IMAGE_REGISTRY=${IMAGE_REGISTRY:-quay.io/medik8s}
+
+# Version for the index image
+export INDEX_VERSION=${INDEX_VERSION:-0.0.1-test}
+# remove leading "v"
+INDEX_VERSION=${INDEX_VERSION//v/}
+
+export NHC_VERSION=${NHC_VERSION:-0.0.1-test}
+NHC_VERSION=${NHC_VERSION//v/}
+
+export SNR_VERSION=${SNR_VERSION:-0.0.1-test}
+SNR_VERSION=${SNR_VERSION//v/}
+
+NHC_GIT_REPO=${NHC_GIT_REPO:-https://github.com/medik8s/node-healthcheck-operator.git}
+NHC_GIT_BRANCH=${NHC_GIT_BRANCH:-main}
+
+SNR_GIT_REPO=${SNR_GIT_REPO:-https://github.com/medik8s/self-node-remediation.git}
+SNR_GIT_BRANCH=${SNR_GIT_BRANCH:-main}
+
+usage() {
+	echo "This script will pull sources of NHC and SNR and build and push all images need for deployment."
+	echo "Finally it will create a CatalogSource, and a Subscription in the openshift-operators namespace to install NHC/SNR."
+	echo "Supported options:"
+	echo "--skip-nhc: skip building NHC, use last built images"
+	echo "--skip-snr: skip building SNR, use last built images"
+	echo "--skip-build: skip building NHC, SNR and index images, use last built images"
+	echo "--skip-deploy: skip deployment"
+	echo "--help | -h: print usage"
+}
+
+BUILD_NHC=true
+BUILD_SNR=true
+BUILD_INDEX=true
+DEPLOY=true
```

Comment:
Might want to assign those value only if they are not pre-assigned.
I think It'll allow a bit more flexibility